docs/data/release_archive.yml

```diff
@@ -428,6 +428,10 @@ release_archive:
       version: 1.0.0
       release_date: 2022-12-21
       filename: "opensearch"
+    - name: "Flink RabbitMQ Connectors"
+      connector: "rabbitmq"
+      version: 3.0.1
+      release_date: 2023-04-13
```

Comment:
Please change the date later, we can add a FIXME notion for release date and then change it when release